net.kvirc.KVIrc.appdata.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018-2020 Flathub maintainers -->
+<component type="desktop">
+  <id>kvirc.desktop</id>
+  <launchable type="desktop-id">net.kvirc.KVIrc</launchable>
+  <project_license>GPL-2.0</project_license>
+  <name>KVIrc</name>
+  <summary>KVIrc is a free portable IRC client</summary>
+  <description>
+    <p>KVIrc is a free portable IRC client based on the excellent Qt GUI toolkit. KVirc is being written by Szymon Stefanek and the KVIrc Development Team with the contribution of many IRC addicted developers around the world.</p>
+  </description>
+  <categories>
+    <category>Communication & News</category>
+  </categories>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://user-images.githubusercontent.com/198756/171932895-b74f5d27-1eaa-4956-9aee-3af191d45b5b.png</image>
+      <image>https://user-images.githubusercontent.com/198756/171932249-de183e57-248d-4a52-bad8-fcd8ee48115d.png</image>
```

Comment:
I don't think this will validate to have multiple `image`. You can have multiple `screenshot` though.

Reply:
You aright, i will change the tag